### ISSUE TITLE
Error of "org-noter: Invalid file path"

### ISSUE THREAD

**Issue by yubrshen** [weirdNox/org-noter#5]
Following the instruction, in a new org-file, with a heading, I did M-x org-noter in this org-file buffer, I provided the path to a PDF file, and I got this error: 
"org-noter: Invalid file path"

I also tried to just use the current directory path without the name of the PDF file. It still didn't work with the same error. 

What should be the path to enter to make it work?

Thanks,

**Comment by weirdNox** [comment]
Hello there! 

This error is thrown when the path you provide is of a directory or from a file you don't have permission to read from (or does not exist). Maybe I'm overlooking some edge case...

Can you provide more info? What OS are you using? Can you provide the directory structure and the path you are pasing to `org-noter`?

**Comment by yubrshen** [comment]
After restarting emacs, I tried to reproduce the problem, but could not. I'll close it for now. 

The only difference from the previous erroneous case might be that this time I entered the correct path to the PDF file the first time. Last time with error, I entered a path to the folder containing the PDF file first, then re-tried to enter the path to the PDF file.

My environment is as follow:
Ubuntun 16.0s LTS
emacs 25.?
spacemacs 0.300 (development)
